App.js

```diff
@@ -76,39 +84,49 @@ export default function App() {
 
   const theme = isDarkTheme ? customDarkTheme : customDefaultTheme;
 
-  // Functions that will nagivate to each componenet // acts like a router
-
-  useEffect(() => {
-    fetchUserData();
-  });
+  useEffect( () => {
+      fetchUserData();
+      fetchFriendsData();
+  }, [isLoggedIn, userId]);
 
   const fetchUserData = () => {
-    setUserData(data.sort((a, b) => (a.name > b.name ? 1 : -1)));
-    // fetch(/*http:<IP HERE>/searchFriends*/)
-    // .then((data) => {
-    //   setUserData(data)
-    // })
-    // setAllUsers() fnc to set all user that exist for friends search
+    console.log('fetchUserData invoked')
+    axios.get(`http://18.219.200.72:8080/user/?user_id=${userId}`)
+    .then(function (response) {
+      setFetchData(response.data)
+    })
+    .catch(function (error) {
+      console.log(error);
+    })
   };
 
+  const fetchFriendsData = () => {
+    console.log('fetchFriendsData invoked')
+      axios.get(`http://18.219.200.72:8080/user/friendsList?user_id=${userId}`)
+      .then(function (response) {
+        setFriendsList( response.data.sort((a, b) => (a.first_name > b.first_name ? 1 : -1)) )
+      })
+      .catch(function (error) {
```

Comment:
Both this and `fetchUserData` have very readable promise structures. Kudos on that!